build-system/tasks/runtime-test.js

```diff
@@ -79,15 +79,15 @@ function getConfig() {
         'SL_Firefox',
         'SL_Safari_11',
         'SL_Edge_17',
-      ]
-      // TODO(amp-infra): Evaluate and add more platforms here.
-      //'SL_Chrome_Dev',
-      //'SL_Chrome_Android_7',
-      //'SL_Safari_12',
-      //'SL_iOS_12'
-      //'SL_iOS_11',
-      //'SL_IE_11'
-      : [
+        'SL_Chrome_Dev',
+        'SL_Firefox_Dev',
```

Comment:
How does adding more browsers affect Travis build running time? 

I wonder if we need Firefox dev. Also Chrome dev is mostly needed as a canary -- we probably only need to run it once in awhile.

Reply:
The time is roughly the same. We're currently running 2 batches for 5 browsers. Adding 2 more still keeps it within 2 batches (batch size = 4).

Sauce Labs doesn't support Chrome Canary, just Dev, so I'd have to look further on how to support that.

Reply:
Sorry I meant "canary" as a early warning signal rather than Chrome Canary.

Reply:
On these lines, I wonder if it makes sense to make the test runs on Chrome Dev and Firefox Dev warning-only, instead of having them fail Travis. (I can think of reasons to do so and not to do so. Your thoughts?)

Reply:
I'm concerned that warnings will go unnoticed but don't like the idea of adding flakiness to builds. What if we default Chrome/Firefox Dev to warnings, but allow tests to "opt in"?